nvflare/private/fed/app/simulator/simulator_worker.py

```diff
@@ -160,7 +161,9 @@ def _create_client(self, args, build_ctx, deploy_args):
 
         client.token = args.token
         self._set_client_status(client, deploy_args, args.simulator_root)
+        start = time.time()
         self._create_client_cell(client, args.root_url, args.parent_url)
+        self.logger.info(f"Complete _create_client_cell.  Time to create client job cell: {time.time() - start}")
```

Comment:
We might want to change this to debug or remove it in the official release so that regular users won't see these messages.

Reply:
changed.